### PR TITLE
Add autosize textarea to chat

### PR DESCRIPTION
**Pull request by maxplt**
# Jira Issue: [IP-70](https://hedvig.atlassian.net/jira/software/projects/IP/boards/19?selectedIssue=IP-70)

## What?
Add TextArea component with autoheight 

## Why?
For comfortable chat using 

## Video
https://user-images.githubusercontent.com/89198006/134462198-35c70f17-a646-4519-af68-55e9038c11e8.mov

## Optional checklist
- [ ] Updated `src/changelog.ts`
- [ ] Codescouted
- [ ] Unit tests written
- [ ] Tested locally

### CODE REVIEW

File: src/features/member/chat/ChatPanel.tsx [hunk 1]
```diff
@@ -19,33 +19,57 @@ const MessagesPanelContainer = styled.div`
 `
 
 const ChatForm = styled.form`
+  position: relative;
   margin: 0 1rem;
   width: 100%;
 `
 const OptionsContainer = styled.div`
+  margin: 0.5em 1em;
+
```
Comment (vonElfvin) [inline src/features/member/chat/ChatPanel.tsx:28]:
```suggestion
```

**Review by vonElfvin — COMMENTED** [review]
Super nice! Very neat package :)

I like that you removed the send button, but a concern I think will come is that people who are doing an ad-hoc chat message via mobile phone cannot send that message 🤔

**Comment by maxplt** [comment]
> I like that you removed the send button, but a concern I think will come is that people who are doing an ad-hoc chat message via mobile phone cannot send that message 🤔

We can show button when screen width < some value

**Review by vonElfvin — APPROVED** [review]
Nice!

**Pull request merged**